<!DOCTYPE html>
<html style="font-size: 16px;" lang="en-US">
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta charset="utf-8">
    <meta name="keywords" content="">
    <meta name="description" content="">
    <meta name="page_type" content="np-template-header-footer-from-plugin">
    <title>ML_background</title>
    <link rel="stylesheet" href="../../../nicepage.css" media="screen">
<link rel="stylesheet" href="ML_background.css" media="screen">
    <script class="u-script" type="text/javascript" src="../../../jquery.js" defer=""></script>
    <script class="u-script" type="text/javascript" src="../../../nicepage.js" defer=""></script>
    <meta name="generator" content="Nicepage 4.2.6, nicepage.com">
    <link id="u-theme-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i|Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i">
    
    
    
    
    
    <script type="application/ld+json">{
		"@context": "http://schema.org",
		"@type": "Organization",
		"name": "Personal Page"
}</script>
    <meta name="theme-color" content="#478ac9">
    <meta property="og:title" content="ML_background">
    <meta property="og:description" content="">
    <meta property="og:type" content="website">
  </head>
  <body class="u-body"><header class="u-clearfix u-header u-header" id="sec-b462"><div class="u-clearfix u-sheet u-sheet-1">
        <nav class="u-menu u-menu-dropdown u-offcanvas u-menu-1">
          <div class="menu-collapse" style="font-size: 1rem; letter-spacing: 0px;">
            <a class="u-button-style u-custom-left-right-menu-spacing u-custom-padding-bottom u-custom-text-active-color u-custom-text-color u-custom-top-bottom-menu-spacing u-nav-link u-text-active-palette-1-base u-text-hover-palette-2-base" href="#">
              <svg viewBox="0 0 24 24"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#menu-hamburger"></use></svg>
              <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><symbol id="menu-hamburger" viewBox="0 0 16 16" style="width: 16px; height: 16px;"><rect y="1" width="16" height="2"></rect><rect y="7" width="16" height="2"></rect><rect y="13" width="16" height="2"></rect>
</symbol>
</defs></svg>
            </a>
          </div>
          <div class="u-custom-menu u-nav-container">
            <ul class="u-nav u-unstyled u-nav-1"><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-custom-color-1 u-text-custom-color-3" href="../../../Index.html">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-custom-color-1 u-text-custom-color-3" href="../../Projects.html">Projects</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-custom-color-1 u-text-custom-color-3" href="../../../RESUME.html">RESUME</a>
</li></ul>
          </div>
          <div class="u-custom-menu u-nav-container-collapse">
            <div class="u-black u-container-style u-inner-container-layout u-opacity u-opacity-95 u-sidenav">
              <div class="u-inner-container-layout u-sidenav-overflow">
                <div class="u-menu-close"></div>
                <ul class="u-align-center u-nav u-popupmenu-items u-unstyled u-nav-2"><li class="u-nav-item"><a class="u-button-style u-nav-link" href="../../../Index.html">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="../../Projects.html">Projects</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="../../../RESUME.html">RESUME</a>
</li></ul>
              </div>
            </div>
            <div class="u-black u-menu-overlay u-opacity u-opacity-70"></div>
          </div>
        </nav>
      </div></header>
    <section class="u-clearfix u-section-1" id="sec-6b00">
      <div class="u-clearfix u-sheet u-sheet-1">
        <a href="ML_method.html" data-page-id="173623253" class="u-border-2 u-border-hover-palette-1-base u-border-palette-1-base u-btn u-btn-round u-button-style u-custom-color-3 u-hover-palette-1-base u-radius-50 u-text-custom-color-1 u-btn-1">Next</a>
        <a href="ML_intro.html" data-page-id="2162906626" class="u-border-2 u-border-hover-palette-1-base u-border-palette-1-base u-btn u-btn-round u-button-style u-custom-color-3 u-hover-palette-1-base u-radius-50 u-text-custom-color-1 u-btn-2">Back</a>
      </div>
    </section>
    <section class="u-clearfix u-section-2" id="sec-ad7c">
      <div class="u-clearfix u-sheet u-valign-middle u-sheet-1">
        <h1 class="u-align-center u-text u-text-custom-color-1 u-text-1"><b><u>Background</u></b>
        </h1>
      </div>
    </section>
    <section class="u-clearfix u-section-3" id="sec-a6c2">
      <div class="u-clearfix u-sheet u-sheet-1">
        <h6 class="u-align-justify u-text u-text-default u-text-1"> DNA is used in crime investigations to solve crimes. This is possible because DNA is both specific and sensitive (1). DNA is sensitive because minute amounts are sufficient enough to identify a person. DNA is on the other hand specific because it is different from one person to another. Thus, whenever we hear in court proceedings that the suspect’s DNA was found at the crime scene, we almost never ask questions. This is mainly due to our strong trust in DNA forensics, as the saying goes, DNA doesn’t lie. This line of thought is however problematic, as forensic DNA analysis is imperfect as it is.&nbsp;<br>
          <br>The results from forensic DNA analysis can sometimes be inaccurate (2,3). These inaccuracies can be caused by problems that occur before, during, or after the analysis. Problems that can occur before the analysis include contamination, degradation, DNA mixing, just to name a few. Problems that can occur during analysis include imperfect analytical techniques that cause artifacts such as drop-ins and drop-outs in the DNA sample. Drop-ins occur when alleles are accidentally added to the DNA sample while drop-outs occur when alleles are omitted.  Post analytical problems can arise as a result of the subjectivity of an analyst and using insufficiently validated forensic DNA software (4).In this project, we focus on studying the problems that occur during DNA analysis. Specifically, studying how the presence of artifacts (unexpected additions or deductions in a DNA sample) can drastically affect a DNA profile. Using machine learning algorithms, we study how the NoC of samples with different properties can be accurately determined. Properties such as the mass (amount of a sample), replicates (reruns of a sample), and height thresholds (specific heights) will be explored.<br>
          <br>&nbsp;In terms of the mass of DNA samples, it is generally believed that samples with low masses have more artifacts (1,5), therefore, it is believed that their determined NoC are sometimes unreliable. The problem however is the definition of a small sample. This is because the definition of a small sample can be different from one jurisdiction to another, from one lab to another, and from one experimental/technical set up to another. We will explore the effect of mass in NoC analysis in this project.&nbsp;<br>
          <br>Furthermore, the differences in replicates will be explored. Replicates are important as they are implemented as a technical fix to the artifact problem. Generally, it has been shown that using replicates can attenuate the effects of replicates (5). By determining the NoC of different replicates of a sample, we believe we can deduce similarities and differences between replicates by comparing the samples they correctly classify or misclassify. Another entity explored in this project is the height threshold of DNA samples. Height threshold is one of the most important parameters used by forensic DNA software. This is because these thresholds are used to determine which alleles to consider or ignore.&nbsp;<br>
          <br>Research has been done on the NoC realm using different approaches(7,8,9). Among such approaches includes the study conducted by Lynch and Cotton in 2018 (6). In their work, Lynch and Cotton built a mathematical framework that explored the NoC of different DNA samples. In their paper, it was stated that samples with high NoC are harder to analyze (especially samples with 3 NoC or more). Their framework was based on an ordered partition algorithm that can list the genotype of samples having NoCs as high as 8. The mathematical framework built by this team is drastically different from the tool created by Swaminathan et. al (8). Swaminathan et. al created a tool called NoCit.&nbsp;<br>
          <br>The NOCit software works by calculating the&nbsp;a posteriori probability&nbsp;(APP) of the number of contributors in the sample. APP is a statistical approach used to determine the probability of an event occurring under certain conditions (8). In the NOCit software, the DNA being analyzed is modeled into a calibrated dataset (made from 35 single-source DNA) and using the continuous model. The result of the NOCit is always given in probabilities, and the number with the highest probability represents the predicted number of contributors. When NOCit was tested on 278 experimental and 40 simulated DNA mixtures, it was found to be accurate in 83% of experimental samples and 85% of the simulated samples.&nbsp;<br>A study conducted by Benschop et.al in 2019 used machine learning algorithms to determine NoC (7). Their model used characteristics such as allele number, heights, frequencies and had 250 “characteristics”. These characteristics were further narrowed down to 19. Using the RepTree algorithm and these 19 characteristics, the highest accuracy attained by this group was 86.6%. This accuracy is quite lower than those attained from a tool called PACE (Probabilistic Assessment for Contributor Estimation) (9). However, it needs less calibration.&nbsp;PACE is currently the most accurate NoC tool. It was built by Marciano et al. in 2017 and uses machine learning algorithms to determine NoC. This tool was created by generating a specific model from a DNA database. In their tests, this group obtained accuracies as high as 95%. The tool, however, is not freely available. As illustrated in the points above, NoC determination has been approached by different labs with varying success.&nbsp;<br>
          <br>The flaw in such approaches, we believe, is that they rely on too much a priori information about the DNA (information about whether the sample is degraded or not, information about the amplification kit used). Some involved a lot of modeling like modeling drop-ins and drop-outs, calculation of degradation, just a name a few. Our approach is just a very simple model. A model that just uses the data present within the DNA profile. We believe it will be interesting to see how such a simple model will compare to the model stated here<br>
          <br>
          <br>
          <br>
          <span style="font-weight: 700;">References</span>
          <br>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <!--[endif]-->
          <span style="font-size: 0.75rem;">“Forensic
DNA Profiling Video Series”, Dan E. Krane, Wright State University, Dayton, OH,&nbsp; <!--[if !supportLists]-->
            <br>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <!--[endif]-->“Denmark to
Review Court Cases Because of Possible DNA Flaw”, New York Times <!--[if !supportLists]-->
            <br>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <!--[endif]-->&nbsp;“Subjectivity and bias in forensic DNA mixture
interpretation”, I.E. Dror, G. Hampikian, Sci. Justice 51 (4) (2011) 204–208. <!--[if !supportLists]-->
            <br>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <!--[endif]-->“Validation
of probabilistic genotyping software for use in forensic DNA casework:
Definitions and illustrations”, Haned et al., Science and Justice 56 (2016)
104–108&nbsp; <!--[if !supportLists]-->
            <br>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <!--[endif]-->“STRmixTM
put to the test: 300 000 non-contributor profiles compared to four-contributor
DNA mixtures and the impact of replicates”, Noel et al., Forensic Science
International: Genetics 41 (2019) 24-31. <!--[if !supportLists]-->
            <br>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <!--[endif]-->“Determination
of the possible number of genotype which can contribute to DNA mixtures:
Non-computer assisted deconvolution should not be attempted for more than two
per son mixtures.” Lynch et al.&nbsp; Forensic
Science International: Genetics 37 (2018) 235-240. <!--[if !supportLists]-->
            <br>7.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <!--[endif]-->“Automated
estimation of the number of contributors in autosomal short tandem repeat
profiles using a machine learning approach”, Benschorp et. al, Forensic Science
International: Genetics 43 (2019) 102150 <!--[if !supportLists]-->
            <br>8.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <!--[endif]-->“NOCIt: A
computational method to infer the number of contributors to DNA samples
analyzed by STR genotyping” Swaminathan et. al,&nbsp;
Forensic Science International: Genetics 16 (2015) 172–180. <br>9.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <!--[endif]-->“PACE:
Probabilistic Assessment for Contributor Estimation— A machine learning-based
assessment of the number of contributors in DNA mixtures” Marciano et. al,
Forensic Science International: Genetics 27 (2017) 82–91 <!--[if !supportLists]-->10.&nbsp; <!--[endif]-->The
WEKA Workbench. Online Appendix for "Data Mining: Practical Machine Learning
Tools and Techniques",Eibe et. al, Morgan Kaufmann, Fourth Edition, 2016.
          </span>
          <br>
        </h6>
      </div>
    </section>
    <section class="u-clearfix u-section-4" id="sec-e401">
      <div class="u-clearfix u-sheet u-sheet-1">
        <a href="ML_method.html" data-page-id="173623253" class="u-border-2 u-border-hover-palette-1-base u-border-palette-1-base u-btn u-btn-round u-button-style u-custom-color-3 u-hover-palette-1-base u-radius-50 u-text-custom-color-1 u-btn-1">Next</a>
        <a href="ML_intro.html" data-page-id="2162906626" class="u-border-2 u-border-hover-palette-1-base u-border-palette-1-base u-btn u-btn-round u-button-style u-custom-color-3 u-hover-palette-1-base u-radius-50 u-text-custom-color-1 u-btn-2">Back</a>
      </div>
    </section>
    
    
    <footer class="u-align-center u-clearfix u-footer u-grey-80 u-footer" id="sec-dd49"><div class="u-clearfix u-sheet u-sheet-1">
        <p class="u-small-text u-text u-text-custom-color-3 u-text-variant u-text-1">Email: mariama10njie@gmail.com</p>
      </div></footer>
  </body>
</html>